modules/svg/register_types.cpp

```diff
@@ -38,7 +39,8 @@ static ImageLoaderSVG *image_loader_svg = nullptr;
 
 void register_svg_types() {
 	tvg::CanvasEngine tvgEngine = tvg::CanvasEngine::Sw;
-	if (tvg::Initializer::init(tvgEngine, 0) != tvg::Result::Success) {
+	int32_t processor_count = OS::get_singleton()->get_processor_count();
+	if (tvg::Initializer::init(tvgEngine, MAX(1, processor_count - 1)) != tvg::Result::Success) {
```

Comment:
As we discussed in PR review meeting, it might be best to use `1` for now, if `0` doesn't work.
Since we already handle multi-threading import of files in parallel, we don't need the thorvg scheduler to also spawn lots of threads for each file. (the `core/os/os.h` include can also be removed then).

Reply:
Updated.